src/MealDisplay.js

```diff
@@ -3,32 +3,36 @@ import './MealDisplay.scss';
 
 function MealDisplay(props) {
 
+    const meal = props.data;
     const ingredients = [];
     for (let i = 1; i <= 30; i++) {
```

Comment:
I think lines 8 - 17 could be cleaned up a bit. A few questions I have:

1) are i = 1, and i <= 30 kind of nice clean numbers, or are those known max numnbers?  It might be cleaner to find out how many ingredients there are:

`const ingredientCount = Object.keys(meal).length;`

2) In general, this might be cleaner as a function written outside the component function.

```
const IngredientList = (ingredientList) => ingredientList.map(() => { return <li>content here</li> })

Reply:
1. Actually the max number is 20, so it should've been i <= 20. Also, keys for the ingredients look like this, so it's a bit hard to just do `Object.keys()`:
<img width="279" alt="Screen Shot 2020-01-22 at 2 28 36 PM" src="https://user-images.githubusercontent.com/16580897/72927671-b13e7980-3d24-11ea-8fa4-eb90718c1649.png">